Support conda-build 3

conda-forge-linter opened the pull request:
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #6.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 3.1.2 and pinning 2018.05.07

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham reviewed (changes requested):
Need to fix `python` stuff and use `compiler` on Windows.

jakirkham committed:
Use new compiler syntax

jakirkham committed:
MNT: Re-rendered with conda-smithy 3.1.2 and pinning 2018.05.07

Now that the `compiler` syntax is used, re-render to update the
feedstock configuration accordingly.

jakirkham committed:
Fix Python location for conda-build 3

As Python is a `build`, not `host`, dependency, correct the path to the
`python` executable for `conda-build` 3.

jakirkham committed:
Require `msinttypes` in `host`

jakirkham committed:
Bump build number to 1

As this is being built with the new compiler packages, bump the build
number to capture package changes that have occurred.
